package.json

```diff
@@ -5,7 +5,12 @@
     "rebuild:browser": "theia rebuild:browser",
     "rebuild:electron": "theia rebuild:electron",
     "start:browser": "yarn rebuild:browser ; yarn --cwd browser-app start",
-    "start:electron": "yarn rebuild:electron ; yarn --cwd electron-app start"
+    "start:electron": "yarn rebuild:electron ; yarn --cwd electron-app start",
+    "download:sample-traces": "git clone https://github.com/tracecompass/tracecompass-test-traces.git; ln -s tracecompass-test-traces/ctf/src/main/resources sample-traces",
+    "download:server": "curl -o trace-compass-server.tar.gz https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/trace-compass-server0.0.1-20200616-0828-linux.gtk.x86_64.tar.gz; tar xf trace-compass-server.tar.gz",
```

Comment:
You can set the link here to be https://download.eclipse.org/tracecompass.incubator/trace-server/rcp/trace-compass-server-latest-linux.gtk.x86_64.tar.gz

It's not working yet, but it will eventually when we fix the CI. Then, as far as I'm concerned, this PR is ready to be merged.

Reply:
done!